src/SFA.DAS.ApprenticeAccounts/Application/Commands/CreateApprenticeAccountCommand/CreateApprenticeAccountCommand.cs

```diff
@@ -6,9 +6,9 @@ namespace SFA.DAS.ApprenticeAccounts.Application.Commands.CreateApprenticeAccoun
     public class CreateApprenticeAccountCommand : IUnitOfWorkCommand
     {
         public Guid ApprenticeId { get; set; }
-        public string FirstName { get; set; } = null!;
-        public string LastName { get; set; } = null!;
-        public string Email { get; set; } = null!;
+        public string? FirstName { get; set; }
```

Comment:
Aren't there nullable warnings if we remove the ` = null!`?

Reply:
No because of string? if we don't makes all these values as nullable. The API generates it's own fluent validation errors and hence the Frontend code had a load of transformations  to convert the autogenerated error messages into meaningful messages for the user (that code has now been removed)

Reply:
Yes, of course!  Ignore me :)